SPECS/prometheus/prometheus.spec

```diff
@@ -4,7 +4,7 @@
 Summary:        Prometheus monitoring system and time series database
 Name:           prometheus
 Version:        2.37.0
-Release:        7%{?dist}
+Release:        8%{?dist}
 License:        ASL 2.0
```

Comment:
Nit: use `Apache-2.0` so this becomes a [valid SPDX expression](https://spdx.org/licenses/).